test/system/notify-liquidated.test.js

```diff
@@ -0,0 +1,167 @@
+describeFn("System -- notify liquidated", () => {
+  const startingBlock = 12368838
+  const tbtcTokenAddress = "0x8daebade922df735c38c80c7ebd708af50815faa"
+  const depositAddress = "0x55d8b1dd88e60d12c81b5479186c15d07555db9d"
+  const bidderAddress = "0xa0216ED2202459068a750bDf74063f677613DA34"
+  const bidderAddress1 = "0xf9e11762d522ea29dd78178c9baf83b7b093aacc"
+  const keepTokenAddress = "0x85Eee30c52B0b379b046Fb0F85F4f3Dc3009aFEC"
+  const tbtcDepositTokenAddress = "0x10b66bd1e3b5a936b7f8dbc5976004311037cdf0"
+  const auctionLength = 86400 // 24h
+  // Only deposits with at least 75% of bonds offered on bond auction will be
+  // accepted by the risk manager.
+  const bondAuctionThreshold = 75
+  // deposit lot size is 5 BTC
+  const lotSize = to1e18(5)
+
+  let tbtcToken
+  let underwriterToken
+  let assetPool
+  let signerBondsSwapStrategy
+  let coveragePool
+  let riskManagerV1
+  let tbtcDeposit
+
+  let governance
+  let rewardsManager
+  let bidder
+  let bidder1
+
+  before(async () => {
+    await resetFork(startingBlock)
+    governance = await ethers.getSigner(0)
+    rewardsManager = await ethers.getSigner(1)
+
+    tbtcToken = await ethers.getContractAt("IERC20", tbtcTokenAddress)
+
+    const UnderwriterToken = await ethers.getContractFactory("UnderwriterToken")
+    underwriterToken = await UnderwriterToken.deploy("Coverage KEEP", "covKEEP")
+    await underwriterToken.deployed()
+
+    const AssetPool = await ethers.getContractFactory("AssetPool")
+    assetPool = await AssetPool.deploy(
+      keepTokenAddress,
+      underwriterToken.address,
+      rewardsManager.address
+    )
+    await assetPool.deployed()
+    await underwriterToken.transferOwnership(assetPool.address)
+
+    const SignerBondsSwapStrategy = await ethers.getContractFactory(
+      "SignerBondsEscrow"
+    )
+    signerBondsSwapStrategy = await SignerBondsSwapStrategy.deploy()
+    await signerBondsSwapStrategy.deployed()
+
+    const Auction = await ethers.getContractFactory("Auction")
+
+    const masterAuction = await Auction.deploy()
+    await masterAuction.deployed()
+
+    const CoveragePool = await ethers.getContractFactory("CoveragePool")
+    coveragePool = await CoveragePool.deploy(assetPool.address)
+    await coveragePool.deployed()
+    await assetPool.transferOwnership(coveragePool.address)
+
+    const RiskManagerV1 = await ethers.getContractFactory("RiskManagerV1")
+    riskManagerV1 = await RiskManagerV1.deploy(
+      tbtcToken.address,
+      tbtcDepositTokenAddress,
+      coveragePool.address,
+      signerBondsSwapStrategy.address,
+      masterAuction.address,
+      auctionLength,
+      bondAuctionThreshold
+    )
+    await riskManagerV1.deployed()
+
+    await coveragePool
+      .connect(governance)
+      .beginRiskManagerApproval(riskManagerV1.address)
+    await increaseTime(2592000) // +30 days
+    await coveragePool
+      .connect(governance)
+      .finalizeRiskManagerApproval(riskManagerV1.address)
```

Comment:
We decided to allow quick risk manager approval in coverage pool:
That's how it will probably be done in real deployments.
If you merged with main, you could replace it with:
```
await coveragePool
  .connect(governance)
  .approveFirstRiskManager(riskManagerV1.address)
```
without adding any delay.

Reply:
Nice 5b800bd